### PR TITLE
Update rules templates for Node 8 compliance

### PR DESCRIPTION
**Pull request by fyockm**
Mostly `new Buffer()` -> `Buffer.from()`, with a few other minor updates

Still retain Node 4 compliance (for now)

### CODE REVIEW

File: redirect-rules/yubico-mfa.md [hunk 1]
```diff
@@ -39,19 +39,20 @@ return function (context, req, res) {
...
-              });
+            };
+            var key = Buffer.from(context.data.yubikey_secret, 'base64');
+            var options = {
+              subject: context.data.user,
+              expiresIn: 60,
+              audience: context.data.yubikey_clientid,
+              issuer: 'urn:auth0:yubikey:mfa'
+            };
+            var token = jwt.sign(payload, key, options]);
```
Comment (thameera) [inline redirect-rules/yubico-mfa.md]: a stray closing bracket?

File: rules/decrypt-sensitive-data.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ function (user, context, callback) {
 
   function decrypt(data) {
     if (!data) {
-      return { };
+      return {};
     }
-    var iv = new Buffer(configuration.ENCRYPT_IV);
+    var iv = Buffer.from(configuration.ENCRYPT_IV, 'base64');
```
Comment (thameera) [inline rules/decrypt-sensitive-data.md]: shouldn't this be `utf8`?

File: redirect-rules/yubico-mfa.md (hunk 1 above)
Comment (fyockm) [reply redirect-rules/yubico-mfa.md]: Good catch!

File: rules/decrypt-sensitive-data.md (hunk 2 above)
Comment (fyockm) [reply rules/decrypt-sensitive-data.md]: Yes, you are correct.

File: rules/decrypt-sensitive-data.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ function (user, context, callback) {
...
     var encodeKey = crypto.createHash('sha256')
     .update(configuration.ENCRYPT_PASSWORD, 'utf-8').digest();
```
Comment (fyockm) [inline rules/decrypt-sensitive-data.md]: this should have been `utf8` (without the dash)

**Pull request merged**